### DISCUSSION TITLE
Jest monorepo has changed its url and no longer matches

### DISCUSSION THREAD

**Discussion by lukeapage** [renovatebot/renovate#24020] [Request Help] [answered]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, select which platform you are using.

Azure DevOps Server

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the problem

This config:
https://docs.renovatebot.com/presets-monorepo/#monorepojest

is no longer correct since it seems jest moved from facebook/jest to jestjs/jest

This causes new updates to open multiple PR's instead of being grouped together.

### Relevant debug logs

N/A

### Have you created a minimal reproduction repository?

I have explained in the description why a minimal reproduction is impossible

**Comment by secustor** [comment, marked as answer]
https://github.com/renovatebot/renovate/pull/24023
